pkg/corerp/frontend/controller/secretstores/kubernetes.go

```diff
@@ -75,8 +78,15 @@ func getOrDefaultEncoding(t datamodel.SecretType, e datamodel.SecretValueEncodin
 	return e, err
 }
 
+// Define a map of required keys for each SecretType
+var requiredKeys = map[datamodel.SecretType][]string{
```

Comment:
This seems to be only referenced from `ValidateAndMutateRequest` function so I'd assume it should be scoped to that. Could you tell me more about the motivation to define it here?

Reply:
moved into function: https://github.com/radius-project/radius/pull/7867